mne/epochs.py

```diff
@@ -498,15 +503,44 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
             Whether to halt program execution until the figure is closed.
             Useful for rejecting bad trials on the fly by clicking on a
             sub plot.
+        n_epochs : int
+            The number of epochs per view on mne_browse_epochs. If trellis is
+            True, this parameter has no effect. Defaults to 20.
+        n_channels : int
+            The number of channels per view on mne_browse_epochs. If trellis is
+            True, this parameter has no effect. Defaults to 10.
+        fig_title : str | None
```

Comment:
can we add this param to the trellis plot to minimize the number of options which are different between the two? Wdyt @jaeilepp ?